plugin/src/main/java/duke/entities/Deadline.java

```diff
@@ -0,0 +1,20 @@
+package duke.entities;
+
+import java.time.LocalDateTime;
+
+import duke.exceptions.DukeException;
+
+/**
+ * Deadline task that has a description and a deadline
+ */
+public class Deadline extends Event {
+    public Deadline(String desc, LocalDateTime deadline) throws DukeException {
+        super(desc, deadline);
+    }
+
+    @Override
+    public String toString() {
+        String marker = isDone() ? "[X] " : "[ ] ";
```

Comment:
I do think using the ternary operator here makes the code much more succinct. However perhaps using if-else loops would be more consistent with the overall design? Furthermore not everyone might agree the former is more readable. I confess this is mostly personal preference though.